Backend/admin/src/components/login/login.css

```diff
@@ -0,0 +1,6 @@
+.login-form {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    margin: auto !important;
```

Comment:
justify is not working ?

Reply:
is working, but i need to set it on his parent which has class `page` so maybe it will affect other pages too

Reply:
i can make a wrapper around it like in questions